minutes/design-meeting/2019-07-08-triage-wg.md

```diff
@@ -0,0 +1,123 @@
+# Triage Working Group Discussion
+
+- [Zulip thread](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/steering.20meeting.202019-07-05)
+
+# Summary
+
+- Discussed ideas of a triage working group and how that might interact
+- Discussed the imminent departure of pnkfelix/nikomatsakis and what to do in the meantime
```

Comment:
Yes, this sentence only caused a minor heart attack.

Reply:
lol

Reply:
```suggestion
- Discussed pnkfelix/nikomatsakis taking leave over next few weeks and what to do in the meantime
```